controls/stig_rhel9/SRG-OS-000324-GPOS-00125.yml

```diff
@@ -2,18 +2,15 @@ controls:
     -   id: SRG-OS-000324-GPOS-00125
         levels:
             - medium
-        title: The operating system must prevent non-privileged users from executing privileged
+        title: {{{ full_name }}} must prevent non-privileged users from executing privileged
             functions to include disabling, circumventing, or altering implemented security
             safeguards/countermeasures.
         rules:
             - disable_ctrlaltdel_burstaction
             - disable_ctrlaltdel_reboot
             - no_tmux_in_shells
             - service_debug-shell_disabled
-            - securetty_root_login_console_only
             - sysctl_fs_protected_hardlinks
             - sysctl_fs_protected_symlinks
-            - sebool_ssh_sysadm_login
-            - selinux_user_login_roles
             - package_sudo_installed
```

Comment:
Cross referencing this SRG with the rules listed here I noticed that in RHEL8 STIG for example, only 
```
            - sysctl_fs_protected_hardlinks
            - sysctl_fs_protected_symlinks
```
have the SRG mapped, other rules don't have it. In RHEL7 STIG there are four items that have this SRG, we may want to double check this. We may want to check if rules here are actually applicable to RHEL8/RHEL9 some of them are enabled in RHEL8/9 content but maybe they don't make sense anymore.

Reply:
this should help us quickly identify rules and their SRG mapping: https://github.com/ComplianceAsCode/content/pull/8493

it wouldn't solve everything though